config/initializers/current_commit_hash.rb

```diff
@@ -1 +1,2 @@
 CURRENT_COMMIT_HASH = `git rev-parse --short HEAD`
+CURRENT_COMMIT_REPO = `git remote get-url origin`
```

Comment:
It might be better to use view helper, instead of global constant.

Reply:
You could be right there but as of the current implementation is using solution like that and i'm not too familiar with the rails framework i would think its better use of time to solve it in refactoring in one go rather then me spending too much time trying to fix problems already present in current code base.